plugins/kotlin/j2k/k1.new/src/org/jetbrains/kotlin/nj2k/printing/JKCodeBuilder.kt

```diff
@@ -64,14 +63,14 @@ internal class JKCodeBuilder(context: NewJ2kConverterContext) {
         }
 
         private fun renderModifiersList(modifiersListOwner: JKModifiersListOwner) {
-            val isOpenByDefault = (modifiersListOwner is JKClass && modifiersListOwner.classKind == INTERFACE) ||
+            val isOpenAndAbstractByDefault = (modifiersListOwner is JKClass && modifiersListOwner.classKind == INTERFACE) ||
                     (modifiersListOwner is JKMethod && modifiersListOwner.parentOfType<JKClass>()?.classKind == INTERFACE)
+                    || (modifiersListOwner is JKField && modifiersListOwner.parentOfType<JKClass>()?.classKind == INTERFACE)
```

Comment:
The initializer is getting complicated, probably it's worth extracting common subexpressions and/or simplifying in other ways.